filters/map_composition.py

```diff
@@ -46,7 +46,8 @@ def responseComplete(self):
         FILES=/path/1.shp;/path/2.shp;/path/3.asc&
         NAMES=Layer 1;Layer 2;Layer 3&
         REMOVEQML=true&
-        OVERWRITE=true
+        OVERWRITE=true&
+        BASEMAP='type=xyz&url=http://tile.osm.org/{z}/{x}/{y}.png?layers=osm;osm'
```

Comment:
I'm wondering what is the `?layers=osm` for?

Reply:
Need to add the simple quote too in the documentation

Reply:
the function of `layers=osm` is to naming the layer as osm. so that we can refer to the layer name later.